README.md

```diff
@@ -6,8 +6,8 @@
 The Flutter plugin for Square [In-App Payments SDK] is a wrapper for the native Android and iOS SDKs and 
 supports the following native In-App Payments SDK versions:
 
-  * iOS: 1.5.7
-  * Android: 1.5.6
+  * iOS: 1.6.0
+  * Android: 1.6.1
```

Comment:
Android should be `1.6.2` and iOS - `1.6.1`, please update these versions across the PR